odoo/http.py

```diff
@@ -1748,7 +1748,7 @@ def _serve_db(self):
                 else:
                     return self._serve_nodb()
             ir_http = self.registry['ir.http']
-            self.env = odoo.api.Environment(cr, self.session.uid, self.session.context)
+            self.env = odoo.api.Environment(cr, self.session.uid, self.session.context | {'lang': 'en_US'})
```

Comment:
Then this one also need to be patched
https://github.com/odoo/odoo/blob/877738df0c760722089799dd732b21b9beb9451f/odoo/addons/base/models/ir_http.py#L161